CHANGES.md

```diff
@@ -1,3 +1,13 @@
+## v2.3.1
+
+### New Features
+
+    - Adaptive moments now calculate the flux of the model rather than a weighted
+      aperture flux.  The flux is a template flux (same method as
```

Comment:
I still don't understand what you mean by this:

```
Adaptive moments now calculate the flux of the model rather than a weighted aperture flux. 
```

Sorry, I am not trying be dense. The flux function below is using the data in the pixels array which is not the model.

```
       if pixel['ierr'] <= 0:
            val = model * oflux
        else:
            val = pixel['val']

        xcorr_sum += model * val * ivar
        msq_sum += model * model * ivar

    flux = xcorr_sum / msq_sum

    res['flux'] = flux
```

I don't see how this quantity differs from index 5 of momsums that admom normally computes.

Reply:
Sorry I should say differs from index 5 of momsums after the correction factors that were put into the code before.

Reply:
A while back admom did return the model normalization iirc but I put in a PR to change that to make it match the other moments fitters. I think what is happening here is that you have refactored the code a bit and correctly identified that I should have made a bigger deal about this than I did.

Reply:
This gives the best fit flux rather than the gaussian aperture weighted flux, it is equivalent to doing a psf flux style template fit.

See results.PSFFluxFitModel

This is the flux needed to properly normalize the model and fill in missing data, and is equivalent to the flux we would get doing the maximum likelihood fit

Reply:
Yeah but I am looking at the old and new code and they appear to compute the same thing.

Reply:
Ohhhh I missed a factor the weight in the denominator. Sorry.